### PR TITLE
Add s3 uploader to pwyb

### PR DESCRIPTION
**Pull request by jamesnunn**
Update repo url to use Data Shed's pywb fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# The Data Shed usage
+
+This repo (and the dependecy pywb) has been forked to make changes to allow uploads of archives and indexes to S3. You will need to set up aws credentials locally. Note the AWS_PROFILE and S3BUCKET environment variables are set.
```
Comment (PsypherPunk) [inline README.md]: _"…the AWS_PROFILE and S3BUCKET environment…"_—should reference `AWS_S3_BUCKET`.

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ ENV PROXY_HOST=localhost \
     DISPLAY=:99 \
     GEOMETRY=1360x1020x16
 
-RUN pip install git+https://github.com/webrecorder/pywb@patch-work
+RUN pip install git+https://github.com/TheDataShed/pywb@add_s3_uploader
```
Comment (PsypherPunk) [inline Dockerfile:17]: Was the `--use-feature=2020-resolver` flag necessary in the end?

**Review by PsypherPunk — CHANGES_REQUESTED** [review]
Could you reduce this to a single commit? Partly to keep it tidy but also, as we're branching off the `patch-work` branch, the fewer commits we have, the fewer we may have to resolve should that branch get merged upstream.

File: Dockerfile (hunk 2 above)
Comment (jamesnunn) [reply Dockerfile:17]: I was going to try and fix the warnings and dependency issues in another branch

**Comment by jamesnunn** [comment]
Just ditching this to create a clean single-commit PR

**Pull request closed without merging**